org.eclipse.xtext/src/org/eclipse/xtext/build/Indexer.java

```diff
@@ -124,7 +123,7 @@ protected static class ResolvedResourceDescription extends AbstractResourceDescr
 
 		private URI uri;
 
-		private ImmutableList<IEObjectDescription> exported;
+		private List<IEObjectDescription> exported;
```

Comment:
am not sure if i like this.

Reply:
If some client code starts calling modifying methods on the `exported` list, it'd be flagged with a deprecation warning iff the type is ImmutableList.

Reply:
To be clear: This is plus. Please keep this as an immutable list.

Reply:
I would understand that if `exported` wouldn't be private, but since it is private, no other code that this class can call it. Anyway I thought it would be an improvement, but since you do not think so, I have reverted the change.